pkg/helpers/conditions/conditions.go

```diff
@@ -27,7 +30,32 @@ func PodContainerRunning(containerName string) watchtools.ConditionFunc {
 		case *corev1.Pod:
 			switch t.Status.Phase {
 			case corev1.PodRunning, corev1.PodPending:
+				for _, s := range t.Status.ContainerStatuses {
+					if s.State.Waiting != nil {
+						// Return error here if pod is pending and container status indicates a failure
+						// otherwise, user will have to wait the timeout period (15 min)
+						// for pod to exit.
+						if s.State.Waiting.Reason == "CreateContainerError" {
+							if s.State.Waiting.Message != "" {
```

Comment:
Drop this condition entirely, just return the error.

Reply:
Add `ImagePullBackOff` condition here and we'll be able to close https://bugzilla.redhat.com/show_bug.cgi?id=1798549 too

Reply:
done, lol not w/out failing for awhile because I was checking for `ImagePullBackoff` not `ImagePullBackOff`  :woman_shrugging: